WordPress/Tests/PHP/YodaConditionsUnitTest.inc

```diff
@@ -84,3 +84,7 @@ return 0 == $foo; // OK
 return $foo == 0; // Bad
 
 if ( (int) $a['interval'] === (int) $b['interval'] ) {} // OK
+
+if ( $GLOBALS['wpdb']->num_rows === 0 ) {} // Bad
+
+if ( $true == strtolower( $check ) ) {} // Bad
```

Comment:
If you want to test function assignment, shouldn't this be one `=` ?

Oh and shouldn't that kind of assignment within a condition be blacklisted as bad practice ? (in another sniff)